README.md

```
@@ -79,13 +79,13 @@ The project initially includes a set of default files created using advanced dat
-### Optional Wordlists and Malware Signatures
+The spellbook has now been fully compressed and is 590MB in size. To download it,
+please check the README page for the updated link. If you are using the complete
+distribution package, you do not need to perform this step.
 
-These files (700MB) can be downloaded using:
+- To install the complete spellbook, simply download and copy it to: /var/spellbook
 
-```bash
-git clone https://github.com/cosmic-zip/witchcraft-wordlists /var/spellbook/
-```
+**link:** https://limewire.com/d/78693105-85b5-4e1f-b8f7-f1a14c598d6b#NS9Qgl0Lc-aofG86s6cBzWYCuV35yY7ErH6YZ4pxSIQ
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)